scripts/instance.template.env

```diff
@@ -41,9 +41,11 @@ APIML_SECURITY_AUTH_PROVIDER=zosmf
 # List of discovery service URLs separated by comma
 ZWE_DISCOVERY_SERVICES_LIST=https://${ZOWE_EXPLORER_HOST}:${DISCOVERY_PORT}/eureka/
 # Enable debug logging for Api Mediation Layer services
-APIML_DEBUG_MODE_ENABLED=false
+APIML_DEBUG_MODE_ENABLED=true
 ####################
 # caching service
+# if true, caching service is deployed on marist along with the rest of API ML and its sanity tests are run
+ZOWE_CACHING_SERVICE_START=true
```

Comment:
This value seems not updated based on the configuration of `zowe_apiml_caching_service_start`. Also if the default value is `true`, then it means it will be turned on by default for all users. Would like to confirm with Jakub what's the best time to default this to be true.

Another question is, if the user has this entry in `instance.env` and is set to `false`. Then later on when we want to switch it to `true`, the upgrade may not work well because it's hard to determine if this value is added by previous migration script, or it's customized by the user already and stay it as `false` is the intention. To avoid we have to switch this false/true back and forth, maybe we don't add this to `instance.env.template`. What we can do is if the user is using Ansible, we modify `playbooks/roles/configure/tasks/configure_instance.yml` to append this value.